interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -422,6 +422,134 @@ def test_equil_results(gas, fuel, ox, Y_Cf, Y_Of, Y_Co, Y_Oo, basis):
 
         test_equil_results(gas, fuel, ox, Y_Cf, Y_Of, Y_Co, Y_Oo, 'mass')
 
+    def test_equivalence_ratio_simple_dilution(self):
+        gas = ct.Solution("gri30.yaml")
+
+        gas.X = "H2:4,O2:1,CO:3,CO2:4,N2:5,CH4:6"
+        state = gas.state
+        self.assertNear(gas.equivalence_ratio(include_species=["H2","O2"]), 2)
+        self.assertNear(gas.equivalence_ratio("H2","O2",include_species=["H2","O2"]), 2)
+        for k in range(state.size):
+            self.assertNear(state[k], gas.state[k])
+
+        def test_simple_dilution(fraction,basis):
+            if isinstance(fraction,str):
+                fractiontype = fraction[:fraction.find(":")]
+            else:
+                fractiontype = list(fraction.keys())[0]
+
+            M_H2 = gas.molecular_weights[gas.species_index("H2")]
+            M_O2 = gas.molecular_weights[gas.species_index("O2")]
+
+            gas.set_equivalence_ratio(2, "H2","O2", fraction=fraction, diluent="CO2", basis=basis)
+            if basis=="mole" and fractiontype=="diluent":
```

Comment:
Cantera uses spaces left and right of comparisons also, i.e.
```python
if basis == "mole" and fractiontype == "diluent":
```
You could also use the more pythonic `fraction_type`.